restic/internal/forget.py

```diff
@@ -32,30 +33,22 @@ def run(restic_base_command,
     if host:
         cmd.extend(['--host', host])
 
-    if keep_last:
-        cmd.extend(['--keep-last', str(keep_last)])
-
-    if keep_hourly:
-        cmd.extend(['--keep-hourly', str(keep_hourly)])
-
-    if keep_daily:
-        cmd.extend(['--keep-daily', str(keep_daily)])
-
-    if keep_weekly:
-        cmd.extend(['--keep-weekly', str(keep_weekly)])
-
-    if keep_monthly:
-        cmd.extend(['--keep-monthly', str(keep_monthly)])
-
-    if keep_yearly:
-        cmd.extend(['--keep-yearly', str(keep_yearly)])
-
-    if keep_within:
-        cmd.extend(['--keep-within', str(keep_within)])
+    _add_keep_option(cmd, '--keep-last', keep_last)
```

Comment:
Only reason I refactored this was to make CI happy, otherwise it said there were too many branches in this function.

Reply:
I turned this off in #193, as I don't think pylint's check here is actually useful. Can we merge that into this PR and swap it back?